### ISSUE TITLE
Convert files to unix line endings

### ISSUE THREAD

**Issue by gemal** [ckeditor/ckeditor-dev#881]
## Are you reporting a feature request or a bug?

From: https://dev.ckeditor.com/ticket/16853

Feature request

some files are still having Windows newlines, so when they are downloaded in other projects they show up as modified even though the files has no changes

files with Windows newlines:
- lang\*.js  (all language files)
- build-config.js 
- CHANGES.md

all other files seems file

**Comment by mlewand** [comment]
Agreed, there's a solid dose of inconsistency across the repo, which should be taken care of.

**Comment by mlocati** [comment]
Yep, nowadays almost any code I see doesn't use Windows line endings. It's time to normalize it, isn't it?
